src/stores/gameList.js

```diff
@@ -52,14 +52,14 @@ export const useGameListStore = defineStore('gameList', {
     joinGame(data) {
       const updatedGame = this.openGames.find((game) => game.id === data.gameId);
       if (updatedGame) {
-        updatedGame.numPlayers = Math.min(2, updatedGame.numPlayers + 1);
+        updatedGame.players.push(data.newPlayer);
         updatedGame.status = data.newStatus;
       }
     },
     otherLeftGame(gameId) {
       const updatedGame = this.openGames.find((game) => game.id === gameId);
-      if (updatedGame) {
-        updatedGame.numPlayers = Math.max(0, updatedGame.numPlayers - 1);
+      if (updatedGame && updatedGame.players.length > 0) {
+        updatedGame.players.pop();
```

Comment:
I'm not 100% on how this whole array works, but will using `.pop()` be an issue it was coming from the second person to join? should we filter out based on username?

Reply:
Yes, I am trying to figure how to remove the specific player. Maybe using pNum

Reply:
You could probably filter out by using the userName on the auth store